deps/build.jl

```diff
@@ -1,7 +1,19 @@
-# If BLAS was compiled with MKL, we want to use MKL for FFTs as well. Thus
-# we have to do this little dance to get around having to use BinDeps for
-# a library that's already linked to Julia.
-if Base.BLAS.vendor() === :mkl
+# If BLAS was compiled with MKL and the user wants MKL-based FFTs, we'll oblige.
+# In that case, we have to do this little dance to get around having to use BinDeps
+# for a library that's already linked to Julia.
+settings = joinpath(@__DIR__, "..", ".build_settings")
+action = ""
+if isfile(settings)
+    provider = readchomp(settings)
+    action = "delete " * normpath(settings)
+elseif haskey(ENV, "JULIA_FFTW_PROVIDER")
```

Comment:
These if statements seem to be in the wrong order to me — the environment variable should override the hidden settings file